tests/Routes/Wiki/ConversionMetricTest.php

```diff
@@ -21,6 +22,8 @@ public function setUp(): void {
         Wiki::query()->delete();
         WikiSiteStats::query()->delete();
         WikiSetting::query()->delete();
+        Carbon::setTestNow(Carbon::parse('first day of October 2023'));
```

Comment:
I've never done this in PHP before, but do we need to "unfreeze" the time again in `tearDown` so that following tests don't get tripped up?

Reply:
right now all tests pass, but I think that is a wise suggestion